mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -557,6 +584,185 @@ class WidgetAdapter(
         }
     }
 
+    class InputViewHolder internal constructor(initData: ViewHolderInitData) :
+        LabeledItemBaseViewHolder(initData, R.layout.widgetlist_inputitem, R.layout.widgetlist_inputitem_compact) {
+        private val inputTextLayout: TextInputLayout = itemView.findViewById(R.id.widgetinput)
+        private val inputText: TextInputEditText = itemView.findViewById(R.id.widgetinputvalue)
+        private var isBinding = false
+        private var hasChanged = false
+        private var showingDatePicker = false
+        private var showingTimePicker = false
```

Comment:
I don't think these are needed. The `FragmentManager` will replace the respective dialogs anyway when a new one is shown. Additionally, as long as the dialog is shown, the widget list won't receive click events.

Reply:
In the emulator, which is slow, I clicked a few times before the picker came up. The result was it came up multiple times. If the FragmentManager replaces and there is no risk, fine to remove.

Reply:
I can reproduce this when removing the booleans and clicking very fast. However, this would probably happen with all other fragments as well (everything that displays bottom sheets), and we never got any complaints about this from users, likely because one can't double touch as fast as one can double click with a mouse.
My suggestion would be removing this logic from the adapter in any case, and handling it in the `FragmentPresenter` (IOW, in `WidgetListFragment`) if needed at some point (that is, when it turns out to be a problem in practice).

Reply:
I will remove this.